sdk/openai/openai/test/utils/createClients.ts

```diff
@@ -20,7 +20,7 @@ export function createClientsAndDeployments(
   switch (apiVersion) {
     case APIVersion.Preview:
     case APIVersion.Stable: {
-      const credential = createTestCredential();
+      const credential = createLiveCredential();
```

Comment:
why not just use `createTestCredential`?

Reply:
I added more context in the PR description.